pyfstat/make_sfts.py

```diff
@@ -770,13 +800,7 @@ def _build_MFD_command_line(self):
         cl_mfd.append('--outSFTdir="{}"'.format(self.outdir))
         cl_mfd.append('--outLabel="{}"'.format(self.label))
 
-        if self.noiseSFTs is not None and self.SFTWindowType is None:
-            raise ValueError(
-                "SFTWindowType is required when using noiseSFTs. "
-                "Please, make sure you understand the window function used "
-                "to produce noiseSFTs."
-            )
-        elif self.noiseSFTs is not None:
```

Comment:
This is removed because MFD does it for you, correct?

Reply:
No, because this strict requirement actually no longer exists. That was part of the v3 changes: the new SFT headers actually contain sufficient info to reconstruct the window and reuse it at injection time. MFD will only throw an error if a) the noiseSFTs are old ones without header window info / have an explicitly "unknown" window in the header or b) the user asks for a window that is mismatched with the one stored in the noiseSFTs header.

https://git.ligo.org/lscsoft/lalsuite/-/merge_requests/2082